api.swagger.yaml

```diff
@@ -2392,172 +2392,3580 @@ paths:
         '404':
-          description: |
-            The notification does not exist.
-      description: Mark a notification as read.
-  /notifications/register/{service_id}/{token}:
-    parameters:
-      - description: Identifying name of the notification service to be used.
-        in: path
-        name: service_id
-        required: true
-        schema:
-          type: string
-          enum:
-            - apns
-            - fcm
-      - description: A token identifying the user to the notification service.
-        in: path
-        name: token
-        required: true
-        schema:
-          type: string
+          description: 'The UUID does not correspond to an existing badge.'
     put:
       tags:
-        - notifications
-      summary: Register a token to identify the user with a push notification service.
+        - 'badges'
+      description: 'Update badge information.'
+      parameters:
+        - description: 'The Invotra UUID of the badge.'
+          in: path
+          name: badge_id
+          required: true
+          schema:
+            type: string
+            format: uuid
       responses:
         '200':
-          description: The token was already registered.
-        '201':
-          description: The token was registered successfully.
+          description: 'The badge information was updated successfully.'
+        '400':
+          description: 'The supplied UUID or JSON was malformed.'
+        '403':
+          description: 'User is not authorized to update this badge.'
         '404':
-          description: Unknown service.
+          description: 'The UUID does not correspond to an existing badge.'
+        '422':
+          description: 'The badge was not updated due to a Drupal validation failure or a JSON schema validation failure.'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/badge-schema-create'
+        required: true
     delete:
       tags:
-        - notifications
-      summary: Deregister a token identifying the user with a push notification service.
+        - 'badges'
+      description: 'Delete a badge.'
+      parameters:
+        - description: 'The Invotra UUID of the badge.'
+          in: path
+          name: badge_id
+          required: true
+          schema:
+            type: string
+            format: uuid
       responses:
         '200':
-          description: The token was deregistered.
+          description: 'The badge was successfully deleted.'
+        '400':
+          description: 'The supplied UUID was malformed.'
+        '403':
+          description: 'User is not authorized to delete this badge.'
         '404':
-          description: The token was not registered or the service was unknown.
-  /feeds/status:
+          description: 'The UUID does not correspond to an existing badge.'
+  '/content_categories':
```

Comment:
All endpoints for this taxonomy are missing the "content_type" property as seen in the UI. Which should be mandatory for POST requests

Reply:
@ParryLuke I've been asking about this one for a while but got no answer. It's a bit tricky:
- It's an array so how will PUTs work for example? If you want to modify the array does the client GET the list manipulate it then PUT it back? Or would you have another endpiont?
-  content_types don't have a UUID (custom ones do, built in have a machine name) example solr :
"sm_field_content_type":["custom_60",
          "advpoll",
          "blog",
          "doc",
          "manual",
          "manual_page",
          "group_status",
          "custom_1574"],

Reply:
added